.chloggen/users_sourabhjain_moreoperationmetrics.yaml

```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: db
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Added `db.cosmosdb.regions_contacted` attribute as dimension for cosmos db metrics.
```

Comment:
It seems it's not set on metrics, is this note accurate?

Reply:
https://github.com/open-telemetry/semantic-conventions/blob/9db67a5b37104dfa06beb9efe7e0974799ffb83c/model/database/common.yaml#L63 added it as common dimension